atomics/T1070.002/T1070.002.yaml

```diff
@@ -57,11 +100,32 @@ atomic_tests:
     This test finds and deletes the system log files within /var/log/ directory using various executions(rm, shred, unlink)
   supported_platforms:
   - macos
+  input_arguments:
+    system_log_name1:
+      description: name or prefix of system log to delete.
+      type: String
+      default: system.log
+    system_log_name2:
+      description: name or prefix of system log to delete.
+      type: String
+      default: system.log
+    system_log_name3:
+      description: name or prefix of system log to delete.
+      type: String
+      default: system.log
+  dependency_executor_name: sh
+  dependencies:
+  - description: |
+      target files must exist
+    prereq_command: |
+      stat /var/log/#{system_log_name1} /var/log/#{system_log_name2} /var/log/#{system_log_name3}
+    get_prereq_command: |
+      touch /var/log/#{system_log_name1} /var/log/#{system_log_name2} /var/log/#{system_log_name3}
   executor:
     command: |
-      sudo find /var/log -name 'system.log.*' -exec rm {} \; #using "rm" execution
-      sudo find /var/log/ -name "system.log.*" -exec shred -u -z -n 3 {} \; #using "shred" execution
-      sudo find /var/log/ -name "system.log.*" -exec unlink {} \; #using "unlink" execution
+      sudo find /var/log -name '#{system_log_name1}.*' -exec rm {} \; #using "rm" execution
+      sudo find /var/log/ -name "#{system_log_name2}.*" -exec shred -u -z -n 3 {} \; #using "shred" execution
+      sudo find /var/log/ -name "#{system_log_name3}.*" -exec unlink {} \; #using "unlink" execution
```

Comment:
these need to be separate files, since the log will be deleted in first command... the subsequent find commands will yield nothing and the shred, unlink will never run.